fcrepo-serialization/src/test/java/org/fcrepo/camel/serialization/RouteTestBinaryEnabled.java

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright 2015 DuraSpace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.camel.serialization;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+import static org.fcrepo.camel.JmsHeaders.EVENT_TYPE;
+import static org.fcrepo.camel.RdfNamespaces.REPOSITORY;
+import org.fcrepo.camel.FcrepoHeaders;
+import org.apache.camel.EndpointInject;
+import org.apache.camel.Exchange;
+import org.apache.camel.Produce;
+import org.apache.camel.ProducerTemplate;
+import org.apache.camel.builder.AdviceWithRouteBuilder;
+import org.apache.camel.component.mock.MockEndpoint;
+import org.apache.camel.test.blueprint.CamelBlueprintTestSupport;
+import org.apache.camel.util.ObjectHelper;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.junit.Test;
+
+/**
+ * Test the route workflow with the 'includeBinaries' config property set to true.
+ *
+ * @author Bethany Seeger
+ * @since 2015-11-24
+ */
+
+public class RouteTestBinaryEnabled extends CamelBlueprintTestSupport {
```

Comment:
Can you factor this and `RouteTest` together? Looks like there are some common methods.